### PR TITLE
Fix the title in the product page for RTL

### PR DESCRIPTION
**Pull request by mermi**
This is ready, please review, I didn't file a bug because it's a small issue

### CODE REVIEW

**Comment by schalkneethling** [comment]
Before:

screen shot 2015-06-24 at 12 13 31

After:

screen shot 2015-06-24 at 12 20 04

Tested in Fx, Chrome, Safari, Opera Next and IE - Thanks @mermi :cake: There is one small nit, and then if you can squash the commits, we will be good to merge o/\o

File: media/css/firefox/family/index.less [hunk 1]
```diff
@@ -244,7 +244,10 @@ html.ios {
     }
 }
 
-
+// Fancy title for RTL
```
Comment (schalkneethling) [inline media/css/firefox/family/index.less]: Perhaps change this to `// Adjust main heading style for RTL` ?

**Comment by schalkneethling** [comment]
Hey there @mermi - If you are having trouble getting Git to play along re: getting this rebased, let me know and I can assist. Thanks for the PR!

**Comment by mermi** [comment]
Hey @schalkneethling somehing strange happen when I run git rebase -i HEAD~3 I can't make the rebase : 
Cannot rebase: You have unstaged changes.
Additionally, your index contains uncommitted changes.
Please commit or stash them.

**Comment by flodolo** [comment]
Leaving the PR open.

@mermi Can you ping me or Schalk on IRC? We should be able to help you fix this

**Comment by flodolo** [comment]
Should be good to merge once the tests pass. @schalkneethling I'll leave the green button to you :-)

**Comment by schalkneethling** [comment]
Yeah! \o/ Thanks @flodolo and @mermi :cake: :coffee:

**Pull request merged**